lang/ca.json

```diff
@@ -2833,6 +2833,7 @@
   "SE0Wpk": "I agree with the <TOSLink>terms of service</TOSLink> of Open Collective",
   "Seanpx": "Required",
   "Search": "Cerca",
+  "search.clear": "Netejar cerca",
```

Comment:
Thanks for making the effort to translate everything! Just so you know, you don't need to do this in the future; these files are auto-generated and will be overridden. The translation work happens on https://crowdin.com/project/opencollective/.